src/app/clusters/administrator-commissioning-server/administrator-commissioning-server.cpp

```diff
@@ -46,36 +46,41 @@ bool emberAfAdministratorCommissioningClusterOpenCommissioningWindowCallback(
     auto & salt                 = commandData.salt;
     auto & passcodeID           = commandData.passcodeID;
 
-    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
+    uint8_t status = 0;
     PASEVerifier verifier;
     const uint8_t * verifierData = pakeVerifier.data();
 
     ChipLogProgress(Zcl, "Received command to open commissioning window");
 
     VerifyOrExit(!Server::GetInstance().GetCommissioningWindowManager().IsCommissioningWindowOpen(),
-                 status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(sizeof(verifier) == pakeVerifier.size(), status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(iterations >= kPBKDFMinimumIterations, status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(iterations <= kPBKDFMaximumIterations, status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(salt.size() >= kPBKDFMinimumSaltLen, status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(salt.size() <= kPBKDFMaximumSaltLen, status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(commissioningTimeout <= kMaxCommissionioningTimeoutSeconds, status = EMBER_ZCL_STATUS_FAILURE);
-    VerifyOrExit(discriminator <= kMaxDiscriminatorValue, status = EMBER_ZCL_STATUS_FAILURE);
+                 status = EmberAfStatusCode::EMBER_ZCL_STATUS_CODE_BUSY);
+    VerifyOrExit(sizeof(verifier) == pakeVerifier.size(), status = EmberAfStatusCode::EMBER_ZCL_STATUS_CODE_PAKE_PARAMETER_ERROR);
```

Comment:
```suggestion
    VerifyOrExit(sizeof(verifier) == pakeVerifier.size(), status = StatusCode::EMBER_ZCL_STATUS_CODE_PAKE_PARAMETER_ERROR);
```
and similar for the others.